pkgs/desktops/gnome/core/evolution-data-server/default.nix

```diff
@@ -101,13 +109,15 @@ stdenv.mkDerivation rec {
     libphonenumber
     boost
     protobuf
-  ] ++ lib.optionals withGtk3 [
-    gtk3
-    webkitgtk_4_1
-  ] ++ lib.optionals withGtk4 [
-    gtk4
-    webkitgtk_5_0
-  ];
+  ] ++ lib.optionals stdenv.isLinux [
+    libaccounts-glib
+  ] ++ lib.optionals stdenv.isDarwin [
+    libiconv
+  ]
+  ++ lib.optional withGtk3 gtk3
```

Comment:
Could you keep the `lib.optionals` so that the style is consistent?

Reply:
Done.